applications/API/tests/test_forms.py

```
@@ -0,0 +1,106 @@
+# applications/API/tests/test_forms.py
+from django.test import TestCase, override_settings
+from django.utils import timezone
+
+from applications.API.forms import InsultReviewForm
+from applications.API.models import Insult, InsultCategory
+
+
+@override_settings(ROOT_URLCONF="applications.API.tests.urls")
+class InsultReviewFormTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `setUpTestData`

Add return type annotation: `None`

(ANN206)